pkg/controller/vsphere/machine_scope_test.go

```diff
@@ -276,11 +282,34 @@ func TestGetCredentialsSecret(t *testing.T) {
 }
 
 func TestPatchMachine(t *testing.T) {
+	g := NewWithT(t)
+
+	configv1.AddToScheme(scheme.Scheme)
+
+	testEnv := &envtest.Environment{
+		CRDDirectoryPaths: []string{filepath.Join("..", "..", "..", "config", "crds")},
```

Comment:
we should probably rely on the CRDs on `install/` and drop the `config` folder. It was originally used for kubemark testing which we don't run anymore.